src/makeCalculator.js

```diff
@@ -4,7 +4,49 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    get result() {
```

Comment:
Remove the getter for 'result' as it is not necessary. The 'result' should be directly accessible as a property of the calculator object.